docs/user/install-cli.md

```diff
@@ -0,0 +1,83 @@
+# Installing the Flight Control CLI
+
+To install `flightctl` on Linux, Mac OS and Windows operating systems you have to download and install the `flightctl` binary file.
+
+## Download archive
+
+1. Click the **❔** (question mark) icon in the upper part of the web console page.
```

Comment:
After this, select "Command Line Tools"  to be taken to the CLI downloads page.

Reply:
added